ballerina/client.bal

```diff
@@ -98,15 +98,25 @@ public isolated client class Client {
     # + orderBy - A comma-separated list of sort keys. Valid keys are 'createdTime', 'folder', 'modifiedByMeTime', 
     #             'modifiedTime', 'name', 'name_natural', 'quotaBytesUsed', 'recency', 'sharedWithMeTime', 'starred', 
     #              and 'viewedByMeTime'
+    # + corpora - Groupings of files to search. Supported values: 'user' (default - My Drive and Shared with me),
+    #             'domain' (domain-wide search), 'allDrives' (all drives where user is a member including shared drives)
     # + return - If successful, stream of files `stream<File>`. Else an `error`
     @display {label: "Get All Files"}
     remote isolated function getAllFiles(@display {label: "Filter String"} string? filterString = (), 
-                                        @display {label: "Order By"} string? orderBy = ()) 
+                                        @display {label: "Order By"} string? orderBy = (),
+                                        @display {label: "Corpora"} string corpora = "user") 
                                 returns @tainted @display {label: "File Stream"} stream<File>|error {
         ListFilesOptional optional = {
             pageSize : 1000,
-            supportsAllDrives : false
```

Comment:
Since `ListFilesOptional` has been used in all the other APIs as well, have we tested whether this breaks any other APIs?